src/chain.ts

```diff
@@ -54,15 +52,15 @@ export function _getPollInterval (
 
 // TODO: extract these definitions
 
-interface Node {
+export interface Node extends Pick<_NodePool, 'getNodeInfo'> {
   api: InstanceType<typeof NodeApi>
+  nodeNetworkId: string
 }
-
 interface Account {
   address: (options: any) => Promise<EncodedData<'ak'>>
 }
 
-type AensName = `${string}.chain`
+export type AensName = `${string}.chain`
```

Comment:
in this PR ae/aens is still in JS, therefore this could maybe be done in that PR.
(Impossible to have in .js files: https://stackoverflow.com/questions/53710368/how-do-i-add-typescript-types-to-a-javascript-module-without-switching-to-typesc)